### ISSUE TITLE
[for new users] All security related issue to be aware currently

### ISSUE THREAD

**Issue by AkechiShiro** [greenshot/greenshot#510]
**Describe the bug**
No clear announcement of security fixes for  
- CVE-2023-34634
- https://greenshot.atlassian.net/browse/SUPPORT-407 (No CVE ID)

**To Reproduce**
Example steps to reproduce the behavior (few scripts for scripts kiddies available) :
- https://www.exploit-db.com/exploits/51633
- https://github.com/rapid7/metasploit-framework/pull/18253
- http://packetstormsecurity.com/files/174222/Greenshot-1.3.274-Deserialization-Command-Execution.html
- Grep security in latest unstable release changelog, sadly nothing will be found.
- Unless you read the code of the latest unstable release, you don't know if CVE-2023-34634 is fixed or not, it's pretty unclear since no one answers on security issue opened on this repo (see additional context)
- It should be fixed by this : https://github.com/greenshot/greenshot/commit/a152e2883fca7f78051b3bd6b1e5cc57355cb44c#diff-bc9ca9a581e30ebf7dac3a58bbdaa8e76b9eaa821f7f618d50ce21e1b8465015R107-R109
- But I didn't confirm as I'm not using this software.

**Expected behavior**
Vulnerabilities shouldn't left unanswered and silently fixed in releases, this sets up a very bad example of how to actually handle security issues, fixing them "silently" doesn't help at all.
Also I strongly suggest a security policy be added here to clearly indicate to whom/how and where security issues should be reported.

If that's too much to ask, then update this FAQ : https://getgreenshot.org/faq/is-greenshot-clean/ and add that CVE/Security issues are not a priority for Greenshot, be transparent at least, that would be hugely helpful.

This software is probably the best for screenshots, sadly the handling of security vulnerabilities isn't great.

**Versions (please complete the following information):**
 - Greenshot version 1.2.10 -> 1.3.277
 - Windows version (not sure if all are affected)

**Additional context**
- [ ] #504 
- [ ] #505 
- [ ] #474

**Comment by AkechiShiro** [comment]
@jklingen Could you please pin this issue on the repo, I believe it is of utter importance that new users are aware of the state of this project.

**Comment by jklingen** [comment]
@AkechiShiro Thanks for the summary.

Just a short notice for now: I agree that our communication on these things could be better, we'll work on this. 
The project currently suffers from lack of spare time, which of course also affects our communication. We need to catch up, step by step.

Please understand that we're still doing all of this in our little spare time, so often things take longer than we'd like.

btw: AFAIK both issues you mentioned are fixed in the latest 1.3 (unstable) release.

**Comment by AkechiShiro** [comment]
Thanks for admitting this failure, I can totally understand, good luck, maintaining open source software is the one of the hardest task on Earth, everyone use it to run the world but very few contributes, I know.

**Comment by aeheck** [comment]
My company just disallowed the use of Greenshot and I am devestated. I have used this software for 10 years now and I don't want to learn other tools. This is a direct result of the security vulnerabilities that haven't been solved for. I asked some colleagues at my former place of work (10k employees) and they had done the same. Sad day.

**Comment by AkechiShiro** [comment]
@aeheck some security issues were fixed in the latest unstable release according to @jklingen, they might have been rolled out to a stable release not sure tho, because I don't use Greenshot

**Comment by derwolle** [comment]
https://greenshot.atlassian.net/browse/SUPPORT-407 seems to be fixed with https://github.com/greenshot/greenshot/releases/tag/v1.3.211

**Comment by derwolle** [comment]
In https://nvd.nist.gov/vuln/detail/CVE-2023-34634 commit https://github.com/greenshot/greenshot/commit/a152e2883fca7f78051b3bd6b1e5cc57355cb44c is referenced as patch, so https://github.com/greenshot/greenshot/releases/tag/v1.3.275 could address CVE-2023-34634 even though not explicitly mentioned.

**Comment by AkechiShiro** [comment]
@derwolle should this issue be closed as there are unstable releases that are patched ?

**Comment by derwolle** [comment]
> @derwolle should this issue be closed as there are unstable releases that are patched and have been there for awhile now, I don't know if I should wait for a stable release or just close it as unstable has been fixed ?

@AkechiShiro I agree to have this issue open until stable is released, but the initial description gave the impression that these bugs are not handled at all (including affected version up to 1.3.277, which is wrong in my opinion). I also agree that the communication of the devs especially regarding vulnerabilities could be better, but they explained themselves severel times (lack of time...).

If some users still want to stick to Greenshots (as I do) and are willing to run an unstable version, they can do this (IMHO) without being fully exposed by the vulnerabilities mentioned in this issue.

**Comment by jklingen** [comment]
The fix is now finally included in the stable (and signed) 1.3 release (as of v1.3.292). 🎉 
(Unstable releases already included it as of v1.3.275.)
